aipcli/flags.go

```diff
@@ -212,7 +212,7 @@ func markRequiredFlags(
 	).([]annotations.FieldBehavior); ok {
 		for _, fieldBehavior := range fieldBehaviors {
 			if fieldBehavior == annotations.FieldBehavior_REQUIRED {
-				_ = cmd.MarkFlagRequired(flag.Name)
+				cmd.MarkFlagsOneRequired(flag.Name, "from-file")
```

Comment:
Is there any way we could refer to the original flag instead of hardcoding the name here? Likely not an issue but would be good to avoid risking changing something elsewhere and forgetting to change it here